### PR TITLE
More robust way of testing lazyloading functionality

### PR DESCRIPTION
**Pull request by 12**
Resolves #NUMBER

**Overall change:** Change how we test lazyloading

**Code changes:**

- Use Cypress `scrollIntoView` method to ensure lazyloaded image has the chance to load, as oppose to `scrollTo('bottom')`, as this may have been running before the page fully rendered.
---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by alex-magana** [comment]
Is there a wait time required to ensure the image is loaded with caption,
or does `visibleImageWithCaption ` solve for this?

**Comment by 12** [comment]
@alex-magana I believe `visibleImageWithCaption` handles this.

File: cypress/integration/newsBodySpec.js [hunk 1]
```diff
@@ -51,25 +51,25 @@ describe('Article Body Tests', () => {
   });
 
   it('should have a visible image with a caption that is lazyloaded and has a noscript fallback image', () => {
-    let thirdFigure;
+    const imageHasNotLoaded = getElement('figure').eq(2);
```
Comment (12) [inline cypress/integration/newsBodySpec.js:54]: I've changed these to two separate `const`'s to make it a tad more readable.

**Review by alex-magana — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**